mease_lab_to_nwb/convert_syntalos/syntalosnwbconverter.py

```diff
@@ -19,22 +55,66 @@ class SyntalosNWBConverter(NWBConverter):
     )
 
     def get_metadata(self):
-        """Auto-populate as much metadata as possible."""
+        metadata = super().get_metadata()
+        metadata['NWBFile'].update(
+            institution="EMBL - Heidelberg",
+            lab="Mease"
+        )
```

Comment:
Why update metadata['NWBFile'] in two different places instead of all at once?

Reply:
Good catch